instrumentation/log4j/log4j-2.7/log4j-2.7.gradle

```diff
@@ -9,7 +9,9 @@ muzzle {
 }
 
 dependencies {
-  library group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.7'
+  compileOnly group: 'org.apache.logging.log4j', name: 'log4j-core', version: '2.7'
```

Comment:
wondering if this change is really needed?

Reply:
Yes, because when we have `library` dependency and running with `latestDepTest=true` its version is replaces with `+` and that breaks tests.

Reply:
ya, i mean the specific change on this line, from `library` to `compileOnly`

Reply:
#1248 @trask @anuraaga